metals/src/main/scala/scala/meta/internal/builds/MavenBuildTool.scala

```diff
@@ -0,0 +1,63 @@
+package scala.meta.internal.builds
+import scala.meta.internal.metals.{MetalsServerConfig, UserConfiguration}
+import scala.meta.io.AbsolutePath
+import scala.tools.nsc.Properties
+
+object MavenBuildTool extends BuildTool {
+
+  private lazy val embeddedMavenLauncher: AbsolutePath = {
+    val mavenWrapper =
+      if (Properties.isWin) "mvnw.cmd"
+      else "mvnw"
+    val out = BuildTool.copyFromResource(tempDir, mavenWrapper)
+    out.toFile.setExecutable(true)
+    Set(
+      s"maven-wrapper.jar",
+      "maven-wrapper.properties",
+      "MavenWrapperDownloader.java"
+    ).foreach { fileName =>
+      BuildTool.copyFromResource(
+        tempDir,
+        s"mvn/wrapper/$fileName",
+        Some(s".mvn/wrapper/$fileName")
+      )
+    }
+    AbsolutePath(out)
+  }
+
+  def args(
+      workspace: AbsolutePath,
+      userConfig: () => UserConfiguration,
+      config: MetalsServerConfig
+  ): List[String] = {
+    // TODO this should have stable maven version
+    // import scala.meta.internal.metals.BuildInfo
+    // val command = s"ch.epfl.scala:maven-bloop_2.10:${BuildInfo.bloopVersion}:bloopInstall"
+    val command =
+      List(
+        s"ch.epfl.scala:maven-bloop_2.10:1.2.5+418-110de48b:bloopInstall",
```

Comment:
We need new bloop release to make it work.

Reply:
What is the workaround for people to try out this PR?

Reply:
Publish newest bloop locally and then replace this version with that bloop version.

Reply:
Alternatively you can use 1.2.5 version, but it will not have sources.